patterns/faq.php

```diff
@@ -2,19 +2,19 @@
 /**
  * Title: Faq
  * Slug: twentytwentyfour/faq
- * Categories: text
+ * Categories: text, about, featured
```

Comment:
I think this could be easier to find if some keywords were added.

Reply:
I added some keywords here, but we can still iterate over this in a new PR.